x-pack/plugins/infra/public/pages/metrics/hosts/hooks/use_control_panels_url_state.ts

```diff
@@ -118,11 +155,13 @@ const mergeDefaultPanelsWithUrlConfig = (dataView: DataView, urlPanels: ControlP
   const visiblePanels = getVisibleControlPanelsConfig(dataView);
 
   // Get list of panel which can be overridden to avoid merging additional config from url
-  const existingKeys = Object.keys(visiblePanels);
+  // and remove helpMessage as it should not be overwritten
+  const existingKeys = Object.keys(visiblePanels).filter((key) => key !== 'helpMessage');
```

Comment:
No need to keep the message in the url as it is defined in the config and it's unlikely to change.